build.gradle

```diff
@@ -485,6 +485,7 @@ project('server') {
         compile group: 'com.google.protobuf', name: 'protobuf-java-util', version: protobufUtilVersion, withoutLogger
         compile group: 'com.google.protobuf', name: 'protobuf-java', version: protobufProtocVersion, withoutLogger
         compile group: 'com.fasterxml.jackson.module', name: 'jackson-module-jsonSchema', version: jacksonVersion, withoutLogger
+        compile group: 'ch.qos.logback', name: 'logback-classic', version: qosLogbackVersion
```

Comment:
nit: we should remove the testCompile dependency below.
`testCompile group: 'ch.qos.logback', name: 'logback-classic', version: qosLogbackVersion`

Reply:
Fixed.